addons/mail/static/tests/discuss/core/attachment_panel_tests.js

```diff
@@ -0,0 +1,46 @@
+/* @odoo-module */
+
+import { click, contains, start, startServer } from "@mail/../tests/helpers/test_utils";
+
+QUnit.module("attachment panel");
+
+QUnit.test("Attachment panel show message when thread has no attachments", async () => {
+    const pyEnv = await startServer();
+    const channelId = await pyEnv["discuss.channel"].create({ name: "General" });
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+
+    await click(".o-mail-Discuss-header button[title='Show Attachment List']");
+    await contains(
+        ".o-mail-Discuss-inspector:contains(This channel doesn't have any attachments.)"
+    );
+});
```

Comment:
- empty newline is useless. Better remove it so it's clearer at a glance these LOCs are grouped together for a single test
- Name of test is misleading. "message" is used a lot to refer to an actual `mail.message`. I would reword to "Empty attachment panel"